src/ts/components/transaction/Send.tsx

```diff
@@ -121,27 +128,31 @@ class Send extends React.Component<ISendProps, ISendState> {
 
   render () {
     if (!this.props.settings.selectedAccount) {
-      return (null)
+      return null
     }
 
     return (
       <ContentContainer>
+        <AccountSection>
+          <Balance address={this.props.settings.selectedAccount.address} />
+        </AccountSection>
+        <div style={{ height: 27 }} />
+        <AccountSection />
         <Form>
-          <Form.Input
-            label='Amount (milli)'
-            onChange={this.changeAmount}
-          />
-          <Form.TextArea
-            label='toAddress'
-            value={this.state.toAddress}
-            onChange={this.changeAddress}
-          />
+          <Amount handleAmountChange={this.changeAmount} handleDigitChange={this.changeSiUnit}/>
+          <div style={{ height: 27 }} />
+          <ToAddress handleAddressChange={this.changeAddress}/>
+          <div style={{ height: 27 }} />
+          <AccountSection>
+            <TxFee>Transaction Fee:
+            <span>  2</span>
```

Comment:
we will replace this hard-coded value, right?

Reply:
yes